content/events/2020-quebeccity/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Québec
+date: '2020-06-18'
+end: '2020-06-18'
+location:
+  country: Canada
+  city: Québec
+  url: https://en.wikipedia.org/wiki/Quebec_City
+  venue: TBD
+#map: bouvet-island-map.png
+sched_slug: kubernetescommunitydayquebeccity
+social:
+  #twitter: 'https://twitter.com/Bouvet_3Y0Z'
+sponsors:
+- level: platine
+  orgs:
+  - name: Microsoft
+    logo: sponsors/microsoft.svg
+    url: https://microsoft.com
+- level: or
+  orgs:
+  - name: RedHat
+    logo: sponsors/redhat.svg
+    url: https://redhat.com
+- level: argent
+  orgs:
+  - name: iA
+    logo: sponsors/ia.png
+    url: https://ia.ca/
+- level: bronze
+  orgs:
+  - name: MayaData
+    logo: 
+    url: https://mayadata.io/
+  - name: Lecentre.net
+    logo: sponsors/lecentrenet.png
+    url: http://www.lecentre.net/
+speakers:
+- name: Phippy
+  image: speakers/phippy-color.png
+  twitter: https://twitter.com/cloudnativefdn
+  bio: Phippy is a simple PHP app, trying to find a home in a cloud native world.
+  homepage: https://phippy.io
+organizers:
+- name: Mathieu Benoit
+  image: organizers/mathieu.jpg
+  homepage: https://www.linkedin.com/in/mathieubenoitqc/
+- name: Sébastien Couturier
+  image: organizers/sebastien-c.jpg
+  #twitter: https://twitter.com/cloudnativefdn
+  homepage: https://www.linkedin.com/in/scouturier/
+- name: Sébastien Thomas
+  image: organizers/sebastien-t.jpg
+  #twitter: https://twitter.com/cloudnativefdn
+  homepage: https://www.linkedin.com/in/prune/
+- name: Martin-Guy Lapointe
+  image: organizers/martin.jpg
+  #twitter: https://twitter.com/cloudnativefdn
+  homepage: https://www.linkedin.com/in/martin-guy-lapointe-37aa37b4/
+- name: Ayrat Khayretdinov
+  image: organizers/ayrat.jpg
+  #twitter: https://twitter.com/cloudnativefdn
+  homepage: https://www.linkedin.com/in/ayrat-khayretdinov-8503b6b/
+organizers_email: organizers-quebec-city@kubernetescommunitydays.org
+local_law_enforcement: 911
+medical_emergency: 911
+cfp_deadline: '2020-05-15'
+sponsor_deadline: '2020-02-15'
```

Comment:
These deadlines seem pretty short for sponsor, and potentially long for the CFP. Will you have time to review proposals and get back to folks for acceptances before event?

Reply:
Thanks for pointing this out, we reviewed those dates with the team and we are comfortable with them for our local edition here. Thanks.